hoomd/mpcd/DryingDropletStreamingMethod.cc

```diff
@@ -69,6 +69,12 @@ void mpcd::DryingDropletStreamingMethod::stream(unsigned int timestep)
      */
     m_geom = std::make_shared<mpcd::detail::SphereGeometry>(end_R, V, m_bc);
 
+    // updating the geometry radius of the filler
+    if (m_filler != NULL)
+        {
+        m_filler->setGeometry(m_geom);
+        }
```

Comment:
You can use a boolean test for if a pointer is not null. I am actually not sure if a shared pointer can ever be equal to `NULL` anyway, as in modern C++ you should use `nullptr`
```suggestion
    if (m_filler)
        {
        m_filler->setGeometry(m_geom);
        }
```